nautobot/docs/release-notes/version-1.3.md

```diff
@@ -151,6 +159,7 @@ As Python 3.6 has reached end-of-life, and many of Nautobot's dependencies have
 ### Added
 
 - [#1226](https://github.com/nautobot/nautobot/issues/1226) - Added custom job intervals package management.
+- [#2125](https://github.com/nautobot/nautobot/issues/2125) - Added `remove_stale_scheduled_jobs` management command which removes all stale scheduled jobs and also added `has_sensitive_variables` field to Job model which prevents the job's input parameters from being saved to the database.
```

Comment:
We also need to add a `### Security` section under 1.3.10 that calls out the potential exposure of sensitive Job inputs to the database, explains the issue and mitigations, etc. Bryan may have suggested wording here.